linux/power_operation_script/set_power_script_default.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check default soft power operation
+# # Parameters:
+#   power_ops: power operation from command vmware-toolbox-cmd
```

Comment:
Please rename `power_ops` to `power_script_op` as it tests one operation at a time, and keep the var name align with `power_script_type`.